abi/abi.go

```diff
@@ -702,3 +702,44 @@ func (c *CertTable) Proto() *pb.CertificateChain {
 		FirmwareCert: firmware,
 	}
 }
+
+// See assembly implementations in cpuid_*.s
+var cpuid func(op uint32) (eax, ebx, ecx, edx uint32)
+
+// SevProduct returns the SEV product enum for the CPU that runs this
+// function. Ought to be called from the client, not the verifier.
+func SevProduct() *pb.SevProduct {
+	eax, _, _, _ := cpuid(1)
```

Comment:
Comments on where this parameter "1" comes from?

Reply:
CPUID[EAX=1] is the processor info that KDS depends on. Comment added.